DEVELOPMENT.md

```diff
@@ -49,25 +49,17 @@ Clone the `mbed-tools` GitHub repository
 git clone git@github.com:ARMmbed/mbed-tools.git
 ```
 
-Setup Pipenv to use Python 3 and install package development dependencies:
+Set up the development environment using tox (tox will create a development environment at mbed-tools/.venv):
 
 ```bash
 cd mbed-tools/
-pipenv --three
-pipenv install --dev --skip-lock
+tox -e dev
+. .venv/bin/activate
```

Comment:
would recommend `source` instead of just the dot, as new developers might think the dot to be a strange typo or something.